src/main.rs

```diff
@@ -48,4 +79,29 @@ fn main() {
             }
         })
         .unwrap();
+}
+
+fn get_config() -> PinConfig {
```

Comment:
We should also implement this in `grpc_main` (just add the config as a parameter to `handle_incoming_telegram` and use it there). Can you add this?

Also, we can move the `PinConfig` and `get_config` stuff in its own file (`pin_config.rs` or something), so that both programs can use it.